src/reporters/json.ts

```diff
@@ -425,6 +423,9 @@ export default class JSONReporter extends BaseReporter {
           await gatherScreenshots(
             join(CACHE_PATH, 'screenshots'),
             async (step, data) => {
+              if (!data) {
```

Comment:
Can we add a note here mentioning that this is needed for fake journeys?

Reply:
I added this just incase, not really related to the commit.